### PR TITLE
Add additional request data to support PPA

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/PAY-2408

### CODE REVIEW

File: app/data/models/payment-request.js [hunk 1]
```diff
@@ -24,7 +25,9 @@ module.exports = (sequelize, DataTypes) => {
     sourceSystem: DataTypes.STRING,
     status: DataTypes.STRING,
     submitted: DataTypes.DATE,
-    value: DataTypes.INTEGER
+    value: DataTypes.INTEGER,
+    debtType: DataTypes.STRING,
+    recoveryDate: DataTypes.STRING
```
Comment (AbidemiAdio) [inline app/data/models/payment-request.js:30]: Consider arranging fields in alphabetical orders

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (AbidemiAdio) [inline package.json:3]: This involves database change, may consider version 1.2.0

Comment (johnwatson484) [reply package.json]: True, does change the behaviour of received requests so guess that's a new feature and therefore minor change.  Updated.

**Review by AbidemiAdio — COMMENTED** [review]
Need to update https://github.com/DEFRA/ffc-diagrams/blob/main/Payments/Statement-Constructor-ERD.drawio with new changes

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**